fw-child/resources/js/map.js

```diff
@@ -560,19 +560,23 @@
       slider.find('.ui-slider-handle').text(ui.value);
 
       let this_pane = slider.attr('data-pane');
-      options.legend.opacity = ui.value / 100;
+
 
       // always set the given pane opacity
       // so it's consistent if we switch sectors
       item
         .find('.leaflet-pane.leaflet-' + this_pane + '-pane')
-        .css('opacity', options.legend.opacity);
+        .css('opacity', ui.value / 100);
 
-      // link legend's opacity
-      item
-        .find('.legend')
-        .find('.legendbox')
-        .css('fill-opacity', options.legend.opacity);
+      if (this_pane == 'raster') {
+        options.legend.opacity = ui.value / 100;
+
+        // link legend's opacity
+        item
+          .find('.legend')
+          .find('.legendbox')
+          .css('fill-opacity', options.legend.opacity);
+      }
```

Comment:
My personal opinion is that changing the opacity of the data should not change the opacity of the legend. In my view, lowering the opacity of the data is not to change its color, it's to see what is below it, so, in that view, the legend doesn't need to be partly invisible.

Reply:
Could we do like in v1 and have an opaque white background behind the legend, so that when we lower its opacity, we don't see the map through the legend?

In v1:
![image](https://github.com/CanadianClimateDataPortal/climatedata-wp-theme/assets/159493515/afd728d2-4e04-4ec5-9429-b9d611a76fdf)

In v2:
![image](https://github.com/CanadianClimateDataPortal/climatedata-wp-theme/assets/159493515/61fad41f-ec80-43d0-8794-90bc83751765)